src/events.js

```diff
@@ -55,11 +56,17 @@ module.exports = function(ctx) {
     currentMode.mousedown(event);
   };
 
+  events.drawChanged = function(event) {
+    featuresChanged = event.features;
+  }
+
   events.mouseup = function(event) {
     mouseDownInfo.isDown = false;
     var target = getFeatureAtAndSetCursors(event, ctx);
     event.featureTarget = target;
 
+    ctx.map.fire('draw.modifiedFeature', {features: featuresChanged});
```

Comment:
We could send it fire the draw.modified change in  mouseDown instead, and then creal featureChanged in mouseUp. I'll experiment with that and see how that feels as an interaction.